reactivesocket-transport-local/src/main/java/io/reactivesocket/transport/local/LocalClient.java

```diff
@@ -0,0 +1,37 @@
+package io.reactivesocket.transport.local;
```

Comment:
AbstractReactiveSocketPerf seems to reference this as "io.reactivesocket.local".   Should we change the import there?  Unless I am missing something.

https://github.com/ReactiveSocket/reactivesocket-java/blob/1.0.x/reactivesocket-examples/src/jmh/java/io/reactivesocket/perf/AbstractReactiveSocketPerf.java